scraper/src/mindtouch2zim/libretexts/errors.py

```diff
@@ -0,0 +1,4 @@
+class BadBookPageError(Exception):
```

Comment:
_BadBookPage_ implies that the Book page as an issue. In this case, the problem is more with the request if I understand correctly. If so, *IncorectBookPage` seems more appropriate.

Reply:
It is a mix of both. Current book page does not have proper structure and/or requests are not appropriate. Not changing since the real problem is unclear.